packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -107,4 +107,11 @@ export interface IChoiceGroupOption {
    */
   labelId?: string;
 
+  /**
+  * Optional input props that will be mixed into the input element, *before* other props are applied. This allows
+  * you to extend the input element with additional attributes, such as data-automation-id needed for automation.
+  * Note that if you provide, for example, "disabled" as well as "inputProps.disabled", the former will take
+  * precedence over the later.
+  */
+  inputProps?: React.HTMLAttributes<HTMLElement | HTMLInputElement>;
```

Comment:
This appears to mirror how we're doing it for checkbox. Seems we should determine the proper way to approach this, and get it into docs (and update checkbox as well)

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts#L63